returnn/datasets/distrib_files.py

```diff
@@ -153,10 +152,10 @@ def __init__(
         :param get_sub_epoch_dataset: callable which returns a dataset dict for a given subset of files
         :param preload_next_n_sub_epochs: how many sub epoch datasets to preload
         :param buffer_size: buffer size for each worker, amount of seqs to prefetch
-        :param distrib_shard_files: set to true to shard the data across worker processes in
-            distributed training scenaria
+        :param distrib_shard_files: deprecated. Replaced by global config option ``dataset_distribution="shard"``.
+
+            Set to true to shard the data across worker processes in distributed training scenaria.
```

Comment:
I don't think this is necessary to mark it as deprecated.
```suggestion
        :param distrib_shard_files: deprecated. set to true to shard the data across worker processes in
            distributed training scenaria
```